x/staking/module.go

```diff
@@ -19,9 +19,6 @@ type AppModule struct {
 	staking.AppModule
 
 	keeper *keeper.Keeper
-
-	// legacySubspace is used solely for migration of x/params managed parameters
-	legacySubspace exported.Subspace
```

Comment:
This exist at cosmos-sdk/x/staking, v0.47.10, yet.

Reply:
But xpla/staking/Appmodule also have a cosmos/staking/AppModule. I think xpla/staking func didn't use legacySubspace, so it's okay to only cosmos/staking/AppModule

Reply:
I see, it's right.